bootstrap/api/transport.go

```diff
@@ -289,8 +289,8 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 			w.WriteHeader(http.StatusBadRequest)
 		case errors.Contains(errorVal, bootstrap.ErrNotFound):
 			w.WriteHeader(http.StatusNotFound)
-		case errors.Contains(errorVal, bootstrap.ErrUnauthorizedAccess):
-			w.WriteHeader(http.StatusForbidden)
+		case errors.Contains(errorVal, bootstrap.ErrUnauthenticated):
+			w.WriteHeader(http.StatusUnauthorized)
```

Comment:
For `ErrUnauthenticated` we return `StatusForbidden`.  The same remark from Authn service for adding `Unauthorized` case applies here.
Not to repeat myself and spam this thread, please take a look at all the other services and consider adding both cases both in the transport layer (`svc/api/transport.go`) and API docs (Swagger files).

Reply:
> @sprql Please don't address my comments yet. @mainflux/maintainers Need to agree on this. We'll do that internally, bring the final decision and let you know. 

@dusanb94  Have you made a decision on this? Should I add this cases?

Reply:
We should probably add these cases, but please don't address this comment yet. We didn't decide what to do with this PR yet, and there is no point that you continue working on this before we give you feedback.

Reply:
@sprql Let's do it like that. Please update the branch, add these cases, and let's review and merge.